pandas/core/arrays/numeric.py

```diff
@@ -130,3 +158,16 @@ def _arith_method(self, other, op):
             )
 
         return self._maybe_mask_result(result, mask, other, op_name)
+
+    def _apply(self, func: Callable, **kwargs) -> "NumericArray":
+        values = self._data[~self._mask]
+        values = np.round(values, **kwargs)
+
+        data = np.zeros(self._data.shape)
+        data[~self._mask] = values
+        return type(self)(data, self._mask)
+
+    @doc(_round_doc)
+    def round(self, decimals: int = 0, *args, **kwargs) -> "NumericArray":
+        nv.validate_round(args, kwargs)
```

Comment:
If we accept args/kwargs here and validate them, then we should also test this (eg doing `np.round(float_arr)` triggers this)